proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,786 @@
+- [x] [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) is the same as `nativeID`.
+- [ ] [`inert`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert).
+- [x] [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) is the same as `accessibilityRole` (with support for web values).
+- [x] `style` is unchanged.
+- [x] [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) is the same as `focusable` (Android, Desktop). Add support for `0` and `-1` values only.
+
+### Event props
+
+Props for commonly handled events should also be supported on core components. Each event handler should receive a W3C event object of the corresponding type, not a synthetic event. These props are merely conveniences and otherwise identical to using the `EventTarget` API mentioned later on. Note that no capture phase props are included, as listening to the capture phase (as well as custom events) should be handled with the `EventTarget` API.
+
+To avoid breaking changes in React Native and React DOM, it may be preferrable to create new event prop names that receive events as specified by web standards, e.g., `on-click`, `on-pointerup`.
+
+- [ ] `onAuxClick` is a `PointerEvent` for auxillary clicks.
+- [ ] `onBlur`.
+- [ ] `onClick` is a `PointerEvent` for primary clicks.
+- [ ] `onContextMenu`.
+- [ ] `onCopy`.
+- [ ] `onCut`.
+- [ ] `onFocus`.
+- [ ] `onGotPointerCapture`.
+- [ ] `onLostPointerCapture`.
+- [ ] `onPaste`.
+- [ ] `onPointerCancel`.
+- [ ] `onPointerDown`.
+- [ ] `onPointerEnter`.
+- [ ] `onPointerLeave`.
+- [ ] `onPointerMove`.
+- [ ] `onPointerOut`.
+- [ ] `onPointerOver`.
+- [ ] `onPointerUp`.
+- [ ] `onKeyDown`.
+- [ ] `onKeyUp`.
+
+Example:
+
+```jsx
+<View
+  aria-hidden={false}
+  aria-label="Accessibility label"
+  id="native-id"
+  on-click={handleDOMClick}
+  on-pointerdown={handlePointerDown}
+  role="button"
+  tabIndex={0}
+>
+```
+
+## `Image` component
+
+Inherits all props and events from "Core Component".
+
+### Primitive props
+
+Various React DOM `img` props have direct equivalents in React Native. For example, `srcSet` is a subset of the `source` prop but without support for specifying headers, etc. Features like alternative text content are also required to target web.
+
+- [x] `alt` prop for alternative text support.
+- [x] `crossOrigin` is equivalent to setting the relevant `Header` in the `source` object. Declaratively configure cross-origin permissions for loaded images.
+- [ ] `decoding`.
+- [ ] `fetchPriority`.
+- [x] `height` is the same as `source.height`.
+- [ ] `loading`.
+- [x] `referrerPolicy` is equivalent to setting the relevant `Header` in the `source` object. Declaratively configure referrer policy.
+- [x] `src` is the same as `source.uri`.
+- [x] `srcSet` is the same as setting a `source` array with `uri` and `scale` properties defined.
```

Comment:
Should RN support `sizes`? https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes

This will allow loading different images based upon device dimensions, a very useful feature when developing for both mobile and desktop

Reply:
Possibly. At Meta we have no examples of that kind of feature being used on either web or native. So for now I think it's more a feature we should attempt to shim in user-space outside of React Native.